packages/examples/cipethernetip/README.md

```diff
@@ -0,0 +1,22 @@
+# Using CIP EtherNet Industrial Protocol as Transport in LIOTA
+
+LIOTA offers CIP EtherNet Industrial protocol as transport at Device end via [cip_ethernet_ip_device_comms](https://github.com/NithyaElango/liota/blob/CipEthernetIP/liota/device_comms/cip_ethernet_ip_device_comms.py)
+
+## Starting the server
+
+To get started with the CIP protocol, a server must be started in any linux machine. Install cpppo in that machine using 
+"sudo pip install cpppo", then to start the server, "python -m cpppo.server.enip -v Scada=DINT[1]"
+
+
+## Using cip_ethernet_ip_device_comms
+
+Initially run the writer program which keeps writing data to the server, check this [simulator](https://github.com/NithyaElango/liota/blob/CipEthernetIP/tests/cipethernetip/cip_ethernet_ip_simulator.py) for the code.
```

Comment:
This piece of code should be checked-in examples folder under CIP. There shouldn't be the reference to any developer branch as we are not sure if the code will exist forever.

Reply:
This simulator code resides under liota/tests, should I move into liota/examples/CIP folder?